python/bootstrap.py

```diff
@@ -321,6 +325,9 @@ def setup_projects(backend):
 if cmd_args.create_taxonomy or create_all:
   setup_taxonomy(backend, lucidfind_collection_id)
 
+if cmd_args.setup_synonyms:
```

Comment:
Do you want them on when we do create_all, too?  That is the default for most of this stuff.